### ISSUE TITLE
[bug] cloak of reflection missing its graphical effects

### ISSUE THREAD

**Issue by bradallred** [gemrb/gemrb#317]
The cloak of reflection is supposed to produce a shiny aura around the wearer ![as seen here](https://baldursgate.fandom.com/wiki/File:Viconia_Cloak_of_Reflection_BG2EE.png)

this used to work as it was one of my test cases for proper effect blending in the subviews branch.

**Comment by lynxlynxlynx** [comment]
Your link doesn't work. Also, do check if you have the fixpack or tweakpack installed, since there is a component in one of them to disable the sphere.

**Comment by bradallred** [comment]
I don't believe I have ever installed any mods here, but I've had this same install for years now. I'll dl a fresh copy from GoG to check.

**Comment by lynxlynxlynx** [comment]
You can just check if you have a weidu.log file. Anyway, I can reproduce it with spell turning.
